src/sql/src/func.rs

```diff
@@ -2260,13 +2268,18 @@ pub static PG_CATALOG_BUILTINS: Lazy<BTreeMap<&'static str, Func>> = Lazy::new(|
                 )
             }) => Int32, 1375;
         },
+        // SQL closely matches PostgreSQL's implementation.
+        // We don't yet support casting to regnamespace, so need to lookup the oid
+        // of 'pg_catalog' via 'pg_namespace'.
         "obj_description" => Scalar {
-            params!(Oid, String) => Operation::binary(|_ecx, _oid, _catalog| {
-                // This function is meant to return the comment on a
-                // database object, but we don't presently support comments,
-                // so stubbed out out to always return NULL.
-                Ok(HirScalarExpr::literal_null(ScalarType::String))
-            }) => String, 1215;
+            params!(Oid, String) => sql_impl_func(
+                "(SELECT description FROM pg_description
+                  WHERE objoid = $1
+                    AND classoid = (
+                      SELECT oid FROM pg_class WHERE relname = $2 AND relnamespace = (
+                        SELECT oid FROM pg_namespace WHERE nspname = 'pg_catalog'))
```

Comment:
The `pg_catalog` OID is now constant across materialize deployments. You can hard-code this in as `mz_pgrepr_consts::oid::SCHEMA_PG_CATALOG_OID` (It's `16657` FWIW).

Reply:
Oh nice, thanks